<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Laptop Sales Manegement</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QGroupBox" name="groupBox">
      <property name="minimumSize">
       <size>
        <width>218</width>
        <height>82</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>218</width>
        <height>82</height>
       </size>
      </property>
      <property name="title">
       <string>User infor</string>
      </property>
      <layout class="QFormLayout" name="formLayout">
       <item row="0" column="0">
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QLabel" name="label">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>22</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>22</height>
            </size>
           </property>
           <property name="text">
            <string>Name:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_2">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>22</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>22</height>
            </size>
           </property>
           <property name="text">
            <string>Position:</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="0" column="1">
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QLabel" name="userName">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>22</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>22</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="userPosition">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>22</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>22</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QGroupBox" name="groupBox_2">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>82</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>82</height>
       </size>
      </property>
      <property name="title">
       <string>Search</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QComboBox" name="cbBoxSearch">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="minimumSize">
            <size>
             <width>70</width>
             <height>22</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>70</width>
             <height>22</height>
            </size>
           </property>
           <item>
            <property name="text">
             <string>Name</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lnSearch">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>22</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>22</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QComboBox" name="cbBoxShow">
           <property name="minimumSize">
            <size>
             <width>70</width>
             <height>22</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>70</width>
             <height>22</height>
            </size>
           </property>
           <item>
            <property name="text">
             <string>All</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Sales</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Purchase</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Preparing</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Delivering</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Completed</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Canceled</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>87</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="btnSearch">
           <property name="minimumSize">
            <size>
             <width>80</width>
             <height>22</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>80</width>
             <height>22</height>
            </size>
           </property>
           <property name="text">
            <string>Search</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="3">
     <widget class="QPushButton" name="btnAddOrder">
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>70</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>70</height>
       </size>
      </property>
      <property name="text">
       <string>Add</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0" rowspan="2">
     <widget class="QTableWidget" name="listDetail">
      <property name="minimumSize">
       <size>
        <width>218</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>218</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::NoSelection</enum>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectRows</enum>
      </property>
      <attribute name="verticalHeaderVisible">
       <bool>false</bool>
      </attribute>
      <attribute name="verticalHeaderHighlightSections">
       <bool>true</bool>
      </attribute>
      <column>
       <property name="text">
        <string>ID</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Item</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Quantity</string>
       </property>
      </column>
     </widget>
    </item>
    <item row="1" column="1" rowspan="2" colspan="2">
     <widget class="QTableWidget" name="listOrder">
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::SingleSelection</enum>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectRows</enum>
      </property>
      <attribute name="verticalHeaderVisible">
       <bool>false</bool>
      </attribute>
      <column>
       <property name="text">
        <string>ID</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Order type</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Name</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Order date</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Completed date</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Total</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Status</string>
       </property>
      </column>
     </widget>
    </item>
    <item row="1" column="3">
     <widget class="QGroupBox" name="groupBox_4">
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>180</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>180</height>
       </size>
      </property>
      <property name="title">
       <string>INGURISHU</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QPushButton" name="btnDetail">
         <property name="text">
          <string>Detail</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnDeliver">
         <property name="text">
          <string>Deliver</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnComplete">
         <property name="text">
          <string>Complete</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnCancel">
         <property name="text">
          <string>Cancel</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnDelete">
         <property name="text">
          <string>Delete</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="2" column="3">
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>295</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>17</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuAccount">
    <property name="title">
     <string>Account</string>
    </property>
    <addaction name="actionLogin"/>
    <addaction name="actionPassword"/>
    <addaction name="actionInfor"/>
   </widget>
   <widget class="QMenu" name="menuManegement">
    <property name="title">
     <string>List</string>
    </property>
    <addaction name="actionLaptop"/>
    <addaction name="separator"/>
    <addaction name="actionCustomer"/>
    <addaction name="actionSupplier"/>
    <addaction name="actionStaff"/>
   </widget>
   <addaction name="menuAccount"/>
   <addaction name="menuManegement"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionLogin">
   <property name="text">
    <string>Logout</string>
   </property>
  </action>
  <action name="actionPassword">
   <property name="text">
    <string>Password</string>
   </property>
  </action>
  <action name="actionLaptop">
   <property name="text">
    <string>Laptop</string>
   </property>
  </action>
  <action name="actionCustomer">
   <property name="text">
    <string>Customer</string>
   </property>
  </action>
  <action name="actionSupplier">
   <property name="text">
    <string>Supplier</string>
   </property>
  </action>
  <action name="actionStaff">
   <property name="text">
    <string>Staff</string>
   </property>
  </action>
  <action name="actionInfor">
   <property name="text">
    <string>Infor</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>cbBoxSearch</tabstop>
  <tabstop>cbBoxShow</tabstop>
  <tabstop>lnSearch</tabstop>
  <tabstop>btnSearch</tabstop>
  <tabstop>btnAddOrder</tabstop>
  <tabstop>btnDetail</tabstop>
  <tabstop>btnDeliver</tabstop>
  <tabstop>btnComplete</tabstop>
  <tabstop>btnCancel</tabstop>
  <tabstop>btnDelete</tabstop>
  <tabstop>listOrder</tabstop>
  <tabstop>listDetail</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
